src/CaptureFileInfo/Manager.cpp

```diff
@@ -33,7 +40,18 @@ void Manager::LoadCaptureFileInfos() {
     settings.setArrayIndex(i);
     QString path = settings.value(kCaptureFileInfoPathKey).toString();
     QDateTime last_used(settings.value(kCaptureFileInfoLastUsedKey).toDateTime());
-    capture_file_infos_.emplace_back(path, std::move(last_used));
+    QDateTime last_modified(settings.value(kCaptureFileInfoLastModifiedKey).toDateTime());
+    uint64_t file_size =
+        static_cast<uint64_t>(settings.value(kCaptureFileInfoFileSizeKey).toULongLong());
```

Comment:
These might also be empty. But on the other hand, it is actually not a problem, because these fields are only used internally. I would be okay to leave this as is